src/index.test.ts

```diff
@@ -275,4 +275,28 @@ new seatsio.EventManager({
 
 new seatsio.SeatingChartDesigner({
     secretKey: 'mySecretKey'
+})
+
+// Seating chart tests
+seatingChart.selectObjects(['A1', { id: 'someId', ticketType: 'aTicketType', amount: 2}])
+seatingChart.deselectObjects(['A1', { id: 'someId', ticketType: 'aTicketType', amount: 2}])
+
+seatingChart.listSelectedObjects().then(objects => {
+    objects.forEach(obj => {
+        obj.accessible
+        obj.category
+        obj.companionSeat
+        obj.deselect
+        obj.displayObjectType
+        obj.id
```

Comment:
Don't think `id` is documented - customers should use `label`.

Reply:
I see, I've removed it.